icinga2/service.go

```diff
@@ -67,7 +68,7 @@ func (s *WebClient) GetService(name string) (Service, error) {
 }
 
 func (s *WebClient) CreateService(service Service) error {
-	serviceCreate := ServiceCreate{Attrs: service}
+	serviceCreate := ServiceCreate{Templates: []string{"generic-template"},Attrs: service}
```

Comment:
Should probably be

```suggestion
	serviceCreate := ServiceCreate{Templates: []string{"generic-service"},Attrs: service}
```

Some quick testing showed that creating a service fails with `500 Internal Server Error - Object could not be created. Error: Import references unknown template: 'generic-template'`, as there's no service template called `generic-template` in our (fairly vanilla) Icinga2 test instance.

Reply:
Changed it